### PR TITLE
Add convenience method for retrieving Creaper OnlineManagementClient

### PR DESCRIPTION
**Pull request by mnovak1**
Add convenience method for retrieving Creaper OnlineManagementClient.

Please make sure your PR meets the following requirements:
- [ ] Pull Request contains description for the changes
- [ ] Pull Request does not include fixes for multiple issues / topics
- [ ] Code compiles and tests are passing
- [ ] Code is self-descriptive and/or documented
- [ ] Description of the tests scenarios is included (see #46)

### CODE REVIEW

File: tooling-server-configuration/src/test/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/CreaperManagementClientTest.java [hunk 1]
```diff
@@ -9,21 +12,30 @@
...
  */
 @RunWith(Arquillian.class)
 public class CreaperManagementClientTest {
 
-    static ArquillianContainerProperties arquillianContainerProperties = new ArquillianContainerProperties(ArquillianDescriptorWrapper.getArquillianDescriptor());
+    static ArquillianContainerProperties arquillianContainerProperties = new ArquillianContainerProperties(
+            ArquillianDescriptorWrapper.getArquillianDescriptor());
+
+    @Test
+    public void testReloadCommandWithDefaultArquillianProperites()
```
Comment (fabiobrz) [inline tooling-server-configuration/src/test/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/CreaperManagementClientTest.java]: Maybe just a little typo here:
"testReloadCommandWithDefaultArquillianProperites" -> "testReloadCommandWithDefaultArquillianProperties"

File: tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/creaper/ManagementClientProvider.java [hunk 2]
```diff
@@ -14,6 +15,17 @@
     private ManagementClientProvider() {
     }
 
+    /**
+     * Creates {@link OnlineManagementClient} for <b>standalone</b> mode, based on {@link ArquillianContainerProperties}
```
Comment (fabiobrz) [inline tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/creaper/ManagementClientProvider.java]: Maybe here you could mention that the descriptor is obtained by static `ArquillianDescriptorWrapper` reference. WDYT?

File: tooling-server-configuration/src/test/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/CreaperManagementClientTest.java (hunk 1 above)
Comment (mnovak1) [reply tooling-server-configuration/src/test/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/CreaperManagementClientTest.java]: Thanks, fixed.

File: tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/creaper/ManagementClientProvider.java (hunk 2 above)
Comment (mnovak1) [reply tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/creaper/ManagementClientProvider.java]: Right, fixed and repushed.

**Comment by mnovak1** [comment]
thanks,merged

**Pull request merged**